### PR TITLE
DF-1849 infobanner candidate-search-profiles.component

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html [hunk 1]
```diff
@@ -14,6 +14,21 @@ <h2 class="pr-5 d-flex">
     </h2>
   </div>
 
+  <alv-closable-alert [notification]="chiscoTransitionNotification">
```
Comment (emina-str) [inline alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html]: think that on the candidate and job search profile, this alert should be on the right side.

File: alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.ts [hunk 2]
```diff
@@ -20,16 +21,23 @@ export class CandidateSearchProfilesComponent implements OnInit {
 
   IconKey = IconKey;
 
+  NotificationType = NotificationType;
```
Comment (emina-str) [inline alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.ts]: Why do we need this?

File: alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html (hunk 1 above)
Comment (emina-str) [reply alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html]: Not quite, it's ok as you've done it under the title. Asked Jonas.

File: alv-portal-ui/src/app/widgets/candidate-search-profiles-widget/candidate-search-profiles-widget.component.html [hunk 3]
```diff
@@ -7,6 +7,21 @@ <h2 class="panel-title">
     {{'portal.candidate-search-profiles.title' | translate}}
   </h2>
 
+  <alv-closable-alert [notification]="chiscoTransitionNotification">
+    <p>Infolge technischer Anpassungen können die Berufsbezeichnungen der
+      Kandidaten-Suchprofile geändert haben. Suchprofile, die auf Berufskategorien
+      zurückgreifen, müssen neu erstellt werden.</p>
+    <p>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/widgets/candidate-search-profiles-widget/candidate-search-profiles-widget.component.html]: Not translated text?

File: alv-portal-ui/src/app/widgets/candidate-search-profiles-widget/candidate-search-profiles-widget.component.html [hunk 4]
```diff
@@ -7,6 +7,21 @@ <h2 class="panel-title">
...
+      <a class="link"
+         href="https://www.arbeit.swiss/secoalv/de/home/menue/unternehmen/stellenmeldepflicht/stellenmeldepflicht-ab-1--januar-2020.html"
+         target="_blank">
+        <fa-icon [icon]="['fas', 'caret-right']"
+                 class="link-icon"></fa-icon>
+        Weitere Informationen
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/widgets/candidate-search-profiles-widget/candidate-search-profiles-widget.component.html]: Not translated text?

File: alv-portal-ui/src/app/job-advertisement/manage-job-ads/manage-job-ad-detail/manage-job-ad-detail.component.html [hunk 5]
```diff
@@ -79,6 +79,20 @@
     </ng-container>
 
     <ng-container class="main-column">
+      <alv-closable-alert [notification]="chiscoTransitionNotification">
+        <p>Infolge technischer Anpassungen können die Berufsbezeichnungen der
+          Kandidaten-Suchprofile geändert haben. Suchprofile, die auf Berufskategorien
+          zurückgreifen, müssen neu erstellt werden.</p>
+        <p>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/manage-job-ads/manage-job-ad-detail/manage-job-ad-detail.component.html]: Not translated text?

File: alv-portal-ui/src/app/job-advertisement/manage-job-ads/manage-job-ad-detail/manage-job-ad-detail.component.html [hunk 6]
```diff
@@ -79,6 +79,20 @@
...
+          <a class="link"
+             href="https://www.arbeit.swiss/secoalv/de/home/menue/unternehmen/stellenmeldepflicht/stellenmeldepflicht-ab-1--januar-2020.html"
+             target="_blank">
+            <fa-icon [icon]="['fas', 'caret-right']"
+                     class="link-icon"></fa-icon>
+            Weitere Informationen
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/manage-job-ads/manage-job-ad-detail/manage-job-ad-detail.component.html]: Not translated text?

File: alv-portal-ui/src/app/job-advertisement/job-ad-search-profiles/job-ad-search-profiles/job-ad-search-profiles.component.html [hunk 7]
```diff
@@ -14,15 +14,29 @@ <h2 class="pr-5 d-flex">
     </h2>
   </div>
 
+  <alv-closable-alert [notification]="chiscoTransitionNotification">
+    <p>Infolge technischer Anpassungen müssen Suchprofile, die auf Berufskategorien
+      zurückgreifen, neu erstellt werden.</p>
+    <p>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search-profiles/job-ad-search-profiles/job-ad-search-profiles.component.html]: Not translated text?

File: alv-portal-ui/src/app/job-advertisement/job-ad-search-profiles/job-ad-search-profiles/job-ad-search-profiles.component.html [hunk 8]
```diff
@@ -14,15 +14,29 @@ <h2 class="pr-5 d-flex">
...
+      <a class="link"
+         href="https://www.arbeit.swiss/secoalv/de/home/menue/unternehmen/stellenmeldepflicht/stellenmeldepflicht-ab-1--januar-2020.html"
+         target="_blank">
+        <fa-icon [icon]="['fas', 'caret-right']"
+                 class="link-icon"></fa-icon>
+        Weitere Informationen
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/job-ad-search-profiles/job-ad-search-profiles/job-ad-search-profiles.component.html]: Not translated text?

File: alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html [hunk 9]
```diff
@@ -14,6 +14,20 @@ <h2 class="pr-5 d-flex">
...
+    <p>Infolge technischer Anpassungen muss die Berufsbezeichnung in der zu duplizierenden
+      Stellenmeldung neu ausgewählt werden.</p>
+    <p>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html]: Not translated text?

File: alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html [hunk 10]
```diff
@@ -14,6 +14,20 @@ <h2 class="pr-5 d-flex">
...
+      <a class="link"
+         href="https://www.arbeit.swiss/secoalv/de/home/menue/unternehmen/stellenmeldepflicht/stellenmeldepflicht-ab-1--januar-2020.html"
+         target="_blank">
+        <fa-icon [icon]="['fas', 'caret-right']"
+                 class="link-icon"></fa-icon>
+        Weitere Informationen
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/candidate/candidate-search-profiles/candidate-search-profiles/candidate-search-profiles.component.html]: Not translated text?

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.component.html [hunk 11]
```diff
@@ -10,6 +10,19 @@
 
   <alv-collapse-panel [panelId]="jobPublicationFormPanelId.JOB_PUBLICATION_OCCUPATION"
                       panelTitle="home.tools.job-publication.occupation.label">
+    <alv-closable-alert [notification]="chiscoTransitionNotification">
+      <p>Die Berufsliste für Stellenmeldungen wurde aktualisiert. Gewisse
+        Berufsbezeichnungen sind nicht mehr verfügbar.</p>
+      <p>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.component.html]: Not translated text?

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.component.html [hunk 12]
```diff
@@ -10,6 +10,19 @@
...
+        <a class="link"
+           href="https://www.arbeit.swiss/secoalv/de/home/menue/unternehmen/stellenmeldepflicht/stellenmeldepflicht-ab-1--januar-2020.html"
+           target="_blank">
+          <fa-icon [icon]="['fas', 'caret-right']"
+                   class="link-icon"></fa-icon>
+          Weitere Informationen
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.component.html]: Not translated text?

File: alv-portal-ui/src/app/widgets/job-search-profiles-widget/job-search-profiles-widget.component.html [hunk 13]
```diff
@@ -7,6 +7,19 @@ <h2 class="panel-title">
     {{'portal.job-ad-search-profiles.title' | translate}}
   </h2>
 
+  <alv-closable-alert [notification]="chiscoTransitionNotification">
+    <p>Infolge technischer Anpassungen müssen Suchprofile, die auf Berufskategorien
+      zurückgreifen, neu erstellt werden.</p>
+    <p>
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/widgets/job-search-profiles-widget/job-search-profiles-widget.component.html]: Not translated text?

File: alv-portal-ui/src/app/widgets/job-search-profiles-widget/job-search-profiles-widget.component.html [hunk 14]
```diff
@@ -7,6 +7,19 @@ <h2 class="panel-title">
...
+      <a class="link"
+         href="https://www.arbeit.swiss/secoalv/de/home/menue/unternehmen/stellenmeldepflicht/stellenmeldepflicht-ab-1--januar-2020.html"
+         target="_blank">
+        <fa-icon [icon]="['fas', 'caret-right']"
+                 class="link-icon"></fa-icon>
+        Weitere Informationen
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/widgets/job-search-profiles-widget/job-search-profiles-widget.component.html]: Not translated text?

File: alv-portal-ui/translations.csv [hunk 15]
```diff
@@ -1164,6 +1165,11 @@ portal.application-documents.document-type.CV,Lebenslauf,Lebenslauf,Curriculum v
 portal.application-documents.document-type.LETTER_OF_APPLICATION,Bewerbungsschreiben,Bewerbungsschreiben,Lettre de motivation,Domande d'impiego,
 portal.application-documents.document-type.TESTIMONIAL,Arbeitszeugnisse,Arbeitszeugnisse,Certificats de travail,Certificati di lavoro,
 portal.application-documents.document-type.CERTIFICATES,Diplome / Zertifikate,Diplome / Zertifikate,Diplômes/certificats,Diplomi / Certificati,
+portal.competence-catalog.infobanners.chisco-transition.candidate-search-profile,Infolge technischer Anpassungen muss die Berufsbezeichnung in der zu duplizierenden Stellenmeldung neu ausgewählt werden.,,,,
```
Comment (emina-str) [inline alv-portal-ui/translations.csv]: the translations are not correct! check the document.

**Pull request merged**